recipes/openmmforcefields/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "openmmforcefields" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmmforcefields/archive/{{ version }}.tar.gz
+  sha256: e6e89a5e75b0a1e1609b7d5480793e6862245cfdad9ae46f0f30a66061480355
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
```

Comment:
```suggestion
skip: True  # [win]

```

Reply:
I think this will work to skip windows builds/testing

Reply:
We can't use platform selectors for skip if `noarch: python` is set. It is expected that Windows will fail for a noarch if dependencies are missing!